### PR TITLE
Pages x Node.js => OpenNext

### PR DESCRIPTION
**Pull request by irvinebroque**
refs https://github.com/opennextjs/docs/pull/3

### CODE REVIEW

File: src/content/docs/pages/framework-guides/nextjs/ssr/get-started.mdx [hunk 1]
```diff
@@ -13,6 +13,12 @@ import { PackageManagers } from "~/components";
 
 Learn how to deploy full-stack (SSR) Next.js apps to Cloudflare Pages.
 
+:::note
+You can now also deploy Next.js apps to [Cloudflare Workers](https://developers.cloudflare.com/workers/frameworks/), including apps that use the Node.js "runtime" from Next.js. This allows you to use the Node.js APIs that Cloudflare Workers provides, and ensures compatibility with a broader set of Next.js features and rendering modes.
```
Comment (tanushree-sharma) [inline src/content/docs/pages/framework-guides/nextjs/ssr/get-started.mdx]:
```suggestion
You can now also deploy Next.js apps to [Cloudflare Workers](https://developers.cloudflare.com/workers/frameworks/), including apps that use the Node.js "runtime" from Next.js. This allows you to use the [Node.js APIs that Cloudflare Workers provides](/workers/runtime-apis/nodejs/#built-in-nodejs-runtime-apis), and ensures compatibility with a broader set of Next.js features and rendering modes.
```

**Review by tanushree-sharma — APPROVED** [review]

File: src/content/docs/pages/framework-guides/nextjs/ssr/get-started.mdx [hunk 2]
```diff
@@ -13,6 +13,12 @@ import { PackageManagers } from "~/components";
 
 Learn how to deploy full-stack (SSR) Next.js apps to Cloudflare Pages.
 
+:::note
+You can now also deploy Next.js apps to [Cloudflare Workers](https://developers.cloudflare.com/workers/frameworks/), including apps that use the Node.js "runtime" from Next.js. This allows you to use the [Node.js APIs that Cloudflare Workers provides](/workers/runtime-apis/nodejs/#built-in-nodejs-runtime-apis), and ensures compatibility with a broader set of Next.js features and rendering modes.
```
Comment (ToriLindsay) [inline src/content/docs/pages/framework-guides/nextjs/ssr/get-started.mdx:17]:
```suggestion
You can also deploy Next.js apps to [Cloudflare Workers](https://developers.cloudflare.com/workers/frameworks/), including apps that use the Node.js "runtime" from Next.js. This allows you to use the [Node.js APIs that Cloudflare Workers provides](/workers/runtime-apis/nodejs/#built-in-nodejs-runtime-apis), and ensures compatibility with a broader set of Next.js features and rendering modes.
```

Comment (ToriLindsay) [reply src/content/docs/pages/framework-guides/nextjs/ssr/get-started.mdx:17]: I've just found that terms like "now" or announcement language tends to age poorly if they're not updated often and I confirmed with another TW that we steer away from that.

File: src/content/docs/pages/framework-guides/nextjs/ssr/get-started.mdx [hunk 3]
```diff
@@ -13,6 +13,12 @@ import { PackageManagers } from "~/components";
...
+
+Refer to the [OpenNext docs for the `@opennextjs/cloudflare` adapter](https://opennext.js.org/cloudflare) to learn how to get started.
```
Comment (ToriLindsay) [inline src/content/docs/pages/framework-guides/nextjs/ssr/get-started.mdx:19]:
```suggestion
Learn how to get started in the [OpenNext documentation for the `@opennextjs/cloudflare` adapter] (https://opennext.js.org/cloudflare)
```

**Review by ToriLindsay — APPROVED** [review]

**Pull request merged**